### PR TITLE
Changed doc for 1.2.x->1.3 upgrade

### LINKED ISSUE #1082: Document revised upgrade procedure for 1.3
Details TBC.

### PR DESCRIPTION
**Pull request by alextopuzov**
Fixes #1082

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/upgrade_appliance.md [hunk 1]
```diff
@@ -78,7 +73,7 @@ Upgrading the vSphere Integrated Containers appliance upgrades vSphere Integrate
 
 **What to Do Next**
 
-- If, in the previous version, you configured vSphere Integrated Containers Registry instances as replication endpoints, upgrade those registry instances. Replication of images from the 1.2.x registry instance to the 1.1.x replication endpoint still functions, but it is recommended that you upgrade the target registry.
+- If, in the previous version, you configured vSphere Integrated Containers Registry instances as replication endpoints, upgrade those registry instances. Replication of images from the 1.3.x registry instance to the 1.2.x replication endpoint still functions, but it is recommended that you upgrade the target registry.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/upgrade_appliance.md]: The next bullet "Due to changes in the data model...." is no longer required.

**Comment by stuclem** [comment]
Looks good. Two additional things:

- The first step in the upgrade_appliance topic should be `1.` (Github renders the first step of a procedure with whatever is specified in the source MD file; Gitbook HTML is OK).
- https://github.com/alextopuzov/vic-product/blob/upgrade13/docs/user_doc/vic_vsphere_admin/upgrading_vic.md includes a link to the deleted data migration topic.

**Comment by stuclem** [comment]
Added @andrewtchin and @morris-jason as reviewers.

**Comment by alextopuzov** [comment]
About: "The first step in the upgrade_appliance topic should be 1. (Github renders the first step of a procedure with whatever is specified in the source MD file; Gitbook HTML is OK)."
It's been like that before my changes, suggested that we do not pay attention to GitHub rendering.

**Comment by stuclem** [comment]
We have had comments about Github rendering in the past. Some people do look at the docs in Github as well as on vmware.github.io. It's a small fix, so I think that we should fix it. You only need to fix the numbering of the first item in the list. The rest will take care of themselves.

**Pull request merged**